### ISSUE TITLE
Best Practice 2 not specific to YAML-LD

### ISSUE THREAD

**Issue by ioggstream** [json-ld/yaml-ld#72]
## Question

While this is reasonable, I think it's not specific to YAML-LD and not related to YAML, and is thus off-topic.
 https://json-ld.github.io/yaml-ld/spec/#prebuilt-contexts

> Do not force users to author contexts
> Instead, provide pre-built contexts that the user can reference by URL for a majority of common use cases

**Comment by gkellogg** [comment]
I agree that these are not specific to YAML-LD, and it is the author who chooses to use a context, not typically the end user (except for the case of re-compating/framing). These can be folded in to the general concepts of [BP1](https://json-ld.github.io/yaml-ld/spec/#use-json-ld-best-practices), which is a living document.

**Comment by anatoly-scherbakov** [comment]
My idea in writing those is the following. If someone writes a JSON-LD document manually (for whatever reason), they probably know what the standard is — and do not have any issues with writing their own `@context`.

In my mind, YAML-LD is a way to include domain experts, students, professionals who are not IT people at all. They might not be interested in diving into the technological intricacies of writing contexts.

And even the IT people might find it nice if the system knows what context to use and doesn't bother them with that copy-paste file per file.

Hence the proposed recommendation.

**Comment by ioggstream** [comment]
@anatoly-scherbakov I think it's worth discussing the above points in a dedicated document providing considerations about context independently on the serialization format.

**Comment by anatoly-scherbakov** [comment]
@ioggstream do you mean moving them into JSON-LD Best Practices document?

**Comment by ioggstream** [comment]
Yes, something unrelated to YAML and instead, related to how managing contexts in different kinds of ecosystems.

**Comment by anatoly-scherbakov** [comment]
I am not sure how related it is to JSON-LD... In fact, if a JSON-LD document is published on the web or if JSON-LD messages are sent from one subsystem to another in a distributed software infrastructure, — then I'd say most likely these documents will contain a `@context` reference.

WDYT?

**Comment by gkellogg** [comment]
Given a new push towards creating a YAML-LD Best Practices as a separate document, I suggest we create a new repository: https://github.com/json-ld/yaml-ld-bp which we can set up with PR-preview, which makes it much easier to see the effect of PRs on the state of the document.
